pkg/infradriver/ssh_infradriver.go

```diff
@@ -274,7 +295,7 @@ func (d *SSHInfraDriver) SetClusterHostAliases(hosts []net.IP) error {
 	for _, host := range hosts {
 		for _, hostAliases := range d.clusterHostAliases {
 			hostname := strings.Join(hostAliases.Hostnames, " ")
-			err := d.CmdAsync(host, shellcommand.CommandSetHostAlias(hostname, hostAliases.IP, "#clusterhostalias-set-by-sealer"))
```

Comment:
do we need to distinguish between the sealer default host alias annotation and the users added ? if use the same one, will delete them all while cluster is running.

Reply:
i think they are same.